### PR TITLE
graduate IngressClassNamespacedParams to GA

### PR DESCRIPTION
**Pull request by hbagdi**
Enhancement issue: https://github.com/kubernetes/enhancements/issues/2365
k/k PR: https://github.com/kubernetes/kubernetes/pull/104636

Code PR hasn't been merged in so
/hold
/milestone 1.23
/sig network

### CODE REVIEW

**Comment by hbagdi** [comment]
Code PR merged.
/hold cancel

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -124,7 +124,8 @@ different Kubernetes components.
 | `IndexedJob` | `false` | Alpha | 1.21 | 1.21 |
 | `IndexedJob` | `true` | Beta | 1.22 | |
 | `IngressClassNamespacedParams` | `false` | Alpha | 1.21 | 1.21 |
-| `IngressClassNamespacedParams` | `true` | Beta | 1.22 | |
+| `IngressClassNamespacedParams` | `true` | Beta | 1.22 | 1.22 |
+| `IngressClassNamespacedParams` | `true` | GA | 1.23 | - |
```
Comment (robscott) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:128]: I think this needs to move down to the "Feature gates for graduated or deprecated features" section.

Comment (hbagdi) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: @robscott updated.

**Comment by robscott** [comment]
Thanks!

/lgtm
/assign @sftim

**Pull request merged**